tests/data/good/article-001.json

```diff
@@ -1,5 +1,12 @@
-[
```

Comment:
IMHO, this file should contain only the data part and not the 'ns' part. The reason for this is, we should have test cases for with and without namespaces.